### ISSUE TITLE
Neural Network for pattern recognition in movies

### ISSUE THREAD

**Issue by Pawan429** [Pawan429/Project-marvel#1]
- [ ] 1. use a Bi-Lstm network
- [ ] 2. input : timestep number in seconds
- [ ] 3. output : sentiment vector of 8 dimensions
- [ ] 4. training data : mix all movies with vectors and timesteps
- [ ] 5. test accuracy against all individual movies. that makes 8 testing accuracies as you took 8 movies.
- [ ] 6. use true positive rate vs false positive rate as indicator of accuracy.
